queries/use-conversation-messages-query.ts

```diff
@@ -88,21 +80,21 @@ export function refetchConversationMessages(
   account: string,
   topic: ConversationTopic
 ) {
-  logger.info("[refetchConversationMessages] refetching messages");
+  logger.debug("[refetchConversationMessages] refetching messages");
   return queryClient.refetchQueries(
     getConversationMessagesQueryOptions(account, topic)
   );
 }
 
-export const addConversationMessage = (args: {
+export const addConversationMessageQuery = (args: {
```

Comment:
I don't think this should be suffixed query, but we should have a way to deliniate direct mutations to the cache vs updates out in the real world. 

maybe optimisticAddConversationMessage?